seedlink_plotter/seedlink_plotter.py

```diff
@@ -252,7 +272,10 @@ def plot_lines(self, stream):
         # single traces of one id together.
         for tr in stream:
             tr.stats.processing = []
-        stream.plot(fig=fig, method="fast", draw=False, equal_scale=False,
+            if len(tr.data) < 5:  # otherwise emty streams make problems with equal scale
+                tr.data = np.array([0.000000000000000000001]) 
```

Comment:
This is a bit weird.. maybe you could drop such ultra short traces instead? `stream.traces = [tr for tr in stream if len(tr) > 5]`?

Reply:
(that would need another `if not stream: ...` check to avoid running into problems trying to plot an empty stream afterwards)